internal/provider/resource_eva_role.go

```diff
@@ -60,11 +60,19 @@ func (t roleProviderType) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Dia
 						},
 						"requires_elevation": {
 							MarkdownDescription: "whether functionality requires elevation or not",
-							Required:            true,
+							Optional:            true,
+							Computed:            true,
 							Type:                types.BoolType,
+							PlanModifiers: []tfsdk.AttributePlanModifier{
+								boolDefaultModifier{
+									Default: false,
+								},
+							},
 						},
 					},
-					tfsdk.ListNestedAttributesOptions{},
+					tfsdk.ListNestedAttributesOptions{
+						MinItems: 1,
```

Comment:
Not sure if its the same but `MinItems` has been removed from: https://github.com/hashicorp/terraform-plugin-go/pull/143

But I might also be referencing the wrong object.

Reply:
I think it's not talking about same struct